captcha/readme.md

```diff
@@ -0,0 +1,21 @@
+# Demo request captcha validator
+Checks the form captcha on '/demo.html'. 
+It only submits to form to Eloqua if the captcha is valid.
+
+
+reCAPTCHA v2 is used. Security preferences (easiest for users - Most secure) and other settings can be configured in the Captcha admin panel:
+https://www.google.com/recaptcha/admin/site/346536634
```

Comment:
Whose account is this recaptcha associated with? Are the creds in lastpass?

Reply:
It's a custom gmail account - hellingerlaurensally@gmail.com
I currently don't have permissions to add items to a shared folder in lastpass.